uupdump.py

```diff
@@ -126,4 +126,4 @@
                 resp = requests.post(URL, headers=headers, data=data)
```

Comment:
Requests is used, re isnt, agreed. But this code badly needs a rewrite. Ill do it sometime later. I wanted to make it working than a clean code.

Reply:
If I were to rate my code for quality I would give it a -1/10

Reply:
lol ok xD, i mean u already imported it on first line

Reply:
I saw that. Thats why I approved and merged .-.